src/components/notification/z-pocket-message/index.tsx

```diff
@@ -0,0 +1,28 @@
+import { Component, h } from "@stencil/core";
+
+/**
+ * @slot  - message body
+ * @slot cta-1 - first CTA
```

Comment:
due cta è un numero fisso? altrimenti si può usare un array come Prop

Reply:
Anche qui, da abstract pare sia così, e i bottoni si posizionano in maniera diversa in base al dispositivo (su una riga uno affianco all'altro per desktop, uno sotto l'altro per mobile), e per fare questo temo di aver bisogno di una struttura fissa. O meglio, questo ci aiuta a non permettere a chi usa il componente di fare quello che vuole.. Boh.. Non so quale sia la soluzione migliore.